plugins/techdocs/src/reader/components/TechDocsReaderPageContent/dom.tsx

```diff
@@ -61,15 +61,13 @@ const MOBILE_MEDIA_QUERY = 'screen and (max-width: 76.1875em)';
 // If a defaultPath is specified then we should navigate to that path replacing the
 // current location in the history. This should only happen on the initial load so
 // navigating to the root of the docs doesn't also redirect.
-const useInitialRedirect = (defaultPath?: string) => {
-  // const hasRun = useRef(false);
-
+export const useInitialRedirect = (defaultPath?: string) => {
   const location = useLocation();
   const navigate = useNavigate();
   const { '*': currPath = '' } = useParams();
 
   useLayoutEffect(() => {
-    if (currPath === '' && defaultPath !== '') {
+    if (currPath === '' && defaultPath && defaultPath !== '') {
```

Comment:
just `&& defaultPath` is enough here. 

`'' == false` is always truthy.

Reply:
Ah, yes. Thank you. My JS is rusty. 🙃